lib/adapters/fs.js

```diff
@@ -14,7 +15,7 @@ module.exports = objectAssign({}, FS, {
 
   writeFile: fs.writeFile,
 
-  mkdir: fs.mkdir,
+  mkdir: mkdirp,
```

Comment:
can you add [fs-extra](https://www.npmjs.com/package/fs-extra) and use [mkdirp](https://github.com/jprichardson/node-fs-extra#mkdirsdir-callback). fs-extra it's a great lib and we should use is against regular 'fs'